### DISCUSSION TITLE
This method allows Kafka to report data. How to disable requests to GRPC

### DISCUSSION THREAD

**Discussion by xhrg** [apache/skywalking#12641] [Q&A] [answered]
This method allows Kafka to report data. How to disable requests to GRPC

`
-javaagent:skywalking-agent.jar -Dskywalking.agent.service_name=kafka-export-demo -Dskywalking.plugin.kafka.bootstrap_servers=ip:9092
`

**Comment by wu-sheng** [comment, marked as answer]
That is still required for keep alive and command pushing.

**Reply by xhrg** [reply]
Thank you

**Comment by wu-sheng** [comment]
Kafka is one way for telemetry data.
